library/public/camera.h

```diff
@@ -9,6 +9,14 @@
 
 namespace f3d
 {
+struct F3D_EXPORT camera_state_t
```

Comment:
You do not need to export header-complete struct/class because no symbol is generated

Reply:
since it is needed for types.h, I assume it may be needed here.